modules/check_time_condition.py

```diff
@@ -0,0 +1,49 @@
+"""
+Checks whether the drone has reached its max flight time and sends it back to launch.
+"""
+import time
+
+from . import evaluate_condition_abstract
+
+
+class CheckTimeCondition(evaluate_condition_abstract.EvaluateCondition):
+    """
+    Checks if drone exceeds the maximum flight time limit, inherits from Evaluate class
+    """
+    def __init__(self, start_time : float, maximum_flight_time: float):
+        """
+        Constructor
```

Comment:
Can we use a `create` method. You can check out repositories like computer-vision-python for examples. It helps with error checking when creating objects.